test/boulder-config-next.json

```diff
@@ -20,7 +20,7 @@
     "shutdownStopTimeout": "10s",
     "shutdownKillTimeout": "1m",
     "subscriberAgreementURL": "http://boulder:4000/terms/v1",
-    "debugAddr": "localhost:8000",
+    "debugAddr": "0.0.0.0:8000",
```

Comment:
@ccppuu let's actually break out the config changes into another PR, since they're not strictly necessary for this one, and they'll conflict with those other changes.